src/lib/operators/join_hash/join_hash_steps.hpp

```diff
@@ -211,11 +211,12 @@ class PosHashTable {
 
 // The bloom filter (with k=1) is used during the materialization and build phases. It contains `true` for each
 // `hash_function(value) % BLOOM_FILTER_SIZE`. Much of how the bloom filter is used in the hash join could be improved:
-// (1) Dynamically check whether bloom filters are worth the memory and computational costs This could be based on the
+// (1) Dynamically check whether bloom filters are worth the memory and computational costs. This could be based on the
 //     input table sizes, the expected cardinalities, the hardware characteristics, or other factors.
 // (2) Choosing an appropriate filter size. 2^20 was experimentally found to be good for TPC-H SF 10, but is certainly
 //     not optimal in every situation.
-// (3) Check whether using multiple hash functions (k>1) brings any improvement.
+// (3) Check whether using multiple hash functions (k>1) brings any improvement. For single-threaded TPC-H SF 10 (which
```

Comment:
`where many are under 2^20`: does this mean that most of these integers have values < 1M?

Reply:
Also, `it however appears as if this has no immediate benefits.`: multiple hash functions have no effect or the check has no effect?

Reply:
> Evaluate whether using multiple hash functions (k>1) brings any improvement. In a first experiment, however, we saw no benefits for single-threaded TPC-H SF 10.

Better? I don't think we need to be too specific here.